oauth/migrations/Version20220211114040.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220211114040 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add "email" and "password" fields to user table';
```

Comment:
Better use backtick(`) symbol for email and password so they are displayed properly in the terminal